cat-theme/src/css/header/_navigation.scss

```diff
@@ -0,0 +1,225 @@
+$background--color:#e6e6e6;
+$icon--color:#1e2023;
+$font--color:#ffffff;
+$font--color--active:#000000;
+$icon--size: 3rem;
+$font--primary:'Fira Sans',
+sans-serif;
+$transition--length: .8;
+
+#navigation {
+    align-items: center;
+    display: flex;
+    position: fixed;
+    justify-content: space-between;
+    width: 100%;
+    z-index: 2;
+}
+
+.site-content {
+    @include flex;
```

Comment:
For a CSS mixin, we should have a line break before/after it to offset it from the rest of the CSS properties to show that it is different.